language/stdlib/modules/libra_configs.move

```diff
@@ -1,36 +1,97 @@
 address 0x0:
 module LibraConfig {
     use 0x0::Transaction;
-    use 0x0::LibraSystem;
+    use 0x0::LibraAccount;
+    use 0x0::LibraTimestamp;
 
     // A generic singleton resource that holds a value of a specific type.
     resource struct T<Config: copyable> { payload: Config }
```

Comment:
What other configs do we envision, other than ValidatorSet? Will discovery information also be part of this?
Will we perhaps want to differentiate between different changes to the config and not always be willing to run `LibraSystem::reconfigure`?

Reply:
DiscoveryInfo feels appropriate here too.

> Will we perhaps want to differentiate between different changes to the config and not always be willing to run LibraSystem::reconfigure?

We discussed this before, it's good to have but doesn't look like must have for V1.